ParkPlanner/Park.cs

```diff
@@ -126,7 +126,7 @@ public partial class Address
         public string City { get; set; }
 
         [JsonProperty("stateCode")]
-        public string StateCode { get; set; }
+        public StateCode StateCode { get; set; }
```

Comment:
Use Enum for State Codes:

Instead of using a string for StateCode in the Address class, consider using an enum to represent state codes. This provides type safety and avoids potential typos.

Reply:
This is a QuickType generated class... and I hesitate to change that.